pkgs/games/osu-lazer/default.nix

```diff
@@ -10,22 +10,29 @@
 , lttng-ust
 , numactl
 , dotnetCorePackages
+, libglvnd
+, xorg
+, udev
 }:
 
 buildDotnetModule rec {
   pname = "osu-lazer";
-  version = "2022.1101.0";
+  version = "2022.1227.0";
```

Comment:
There is the `2022.1228.0` bugfix release, so I think we can change to it.

Reply:
done in 935bed0834734bc59ca1b04b3405edec51e853cb